go/libkb/secret_store_file_test.go

```diff
@@ -42,6 +42,10 @@ func testSSDir(t *testing.T) (string, func()) {
 }
 
 func TestSecretStoreFileRetrieveSecret(t *testing.T) {
+	tc := SetupTest(t, "SecretStoreFile", 1)
+	defer tc.Cleanup()
+	m := NewMetaContextForTest(tc)
```

Comment:
i don't think entire test structure needs to be initialized for this test. or did it not work without it?

Reply:
It didn't work because of the added `TraceTimed`s, they need G

Reply:
got it, thanks